python-package/securexgboost/core.py

```diff
@@ -1560,11 +1652,22 @@ def predict(self, data, output_margin=False, ntree_limit=0, pred_leaf=False,
             When this is True, validate that the Booster's and data's feature_names are identical.
             Otherwise, it is assumed that the feature_names are the same.
 
+        username: string
+            Use the name to find the encryption key used for prediction result
+
+        decrypt: bool
+            When this is True, the predictions received from the enclave are decrypted using the user's symmetric key
+
         Returns
         -------
         prediction : numpy array
         num_preds: number of predictions
         """
+        # check the global variable for current_user
+        if username is None and "current_user" in globals():
+            username = globals()["current_user"]
+        if username is None:
```

Comment:
Do we still need this check?

Reply:
Perhaps we should add this check to all functions that use the globals (and tell the user to invoke `init_user` if required)?

Reply:
(To be addressed in a separate PR.)